app/models/rock_and_pebble.rb

```diff
@@ -1,5 +1,13 @@
 class RockAndPebble < ApplicationRecord
     belongs_to :rock, class_name: 'User'
     belongs_to :pebble, class_name: 'User'
-end
 
+    def rock_pebble_info(reason = nil )
+      {rock_name: rock.name,
+       pebble_name: pebble.name,
+       pebble_email: pebble.email,
+       rock_email: rock.email,
+       reason: reason
+      }
```

Comment:
I see what you did here, as a way to package up all the info; I'm wondering if, instead of passing in `reason` in order to add it to the hash, would it introduce less coupling if this method just returned the four things it knows (names, emails) and let the mailer add the reason key/value pair to the hash?

Reply:
Did my best to clean this up as you suggested. Let me know if the changes don't reflect your thoughts.

Reply:
Looks good!